### PR TITLE
fix #79097 koreatimes.co.kr

### LINKED ISSUE #79097: koreatimes.co.kr
### Issue URL (Social Widget)
 [https://www.koreatimes.co.kr/www/tech/2021/04/133_306481.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.koreatimes.co.kr%2Fwww%2Ftech%2F2021%2F04%2F133_306481.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | EdgeChromium
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> EasyList Italy

### PR DESCRIPTION
**Pull request by wout-codes**
#79097

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -39,6 +39,7 @@ donbosco-medien.de,giantbomb.com,strefainwestorow.pl,narcity.com,lingohut.com,mt
...
 !
 livingetc.com,gardeningetc.com##.social__item--newsletter
+koreatimes.co.kr,m.chosun.com,sports.donga.com##.sns_area
```
Comment (piquark6046) [inline SocialFilter/sections/specific.txt:42]: What about this?
```suggestion
koreatimes.co.kr,m.chosun.com,sports.donga.com##div.sns_area
```

Comment (wout-codes) [reply SocialFilter/sections/specific.txt:42]: Isn't it the same thing?

Comment (piquark6046) [reply SocialFilter/sections/specific.txt:42]: Yes, but I think It is safer than yours.

Comment (wout-codes) [reply SocialFilter/sections/specific.txt]: Okay.

**Review by ghost — APPROVED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply SocialFilter/sections/specific.txt]: What is broken by `##.sns_area`?

Comment (wout-codes) [reply SocialFilter/sections/specific.txt]: I don't know too.

**Pull request merged**